src/environments/utils.py

```diff
@@ -31,4 +33,30 @@ def _set_xform_orientation(xform, orientation):
         q = Gf.Quatd(orientation[3], Gf.Vec3d(orientation[0], orientation[1], orientation[2]))
     else:
         q = Gf.Quatf(orientation[3], Gf.Vec3f(orientation[0], orientation[1], orientation[2]))
-    orient_op.Set(q)
\ No newline at end of file
+    orient_op.Set(q)
+
+def transform_orientation_into_xyz(orientation_wxyz):
+    """
+    This function is used for transforming prim's orientation, that is retrieved from the simulation, 
+    from [w, x, y, z] format into [x, y, z] (Euler transformation) which is the forat used to display orientation in IsaacSim.
```

Comment:
minor: 'which' next to xyz makes it confusing which format is used for orientation in isaacsim. please adjust
please add the link to the isaacsim doc about the format